src/mjolnir/speed_assigner.h

```diff
@@ -263,9 +262,7 @@ class SpeedAssigner {
     if (directededge.link()) {
       uint32_t speed = directededge.speed();
       Use use = directededge.use();
-      if (use == Use::kTurnChannel && infer_turn_channels) {
-        speed = static_cast<uint32_t>((speed * kTurnChannelFactor) + 0.5f);
```

Comment:
Should be noticed that durations will be increased after removing this line so I'm hesitating is it the right move..
Alternative safe solution might be
- leave speed_assigner code as it is
- handle only `if (edge.has_historic_speeds)` case in the `AutoCost::EdgeCost`

cons:
- Different turn channel factor in 2 places. It would be better to have everything in one place

Reply:
thats a very good question you raise... here's my opinion:

in general, our speed assignments are always too optimistic. also turn channels are by definition short. i think its maybe its not a big deal to get 20% slower considering these two facts..

Reply:
> also turn channels are by definition short. i think its maybe its not a big deal to get 20% slower considering these two facts

agreed